modules/@apostrophecms/rich-text-widget/ui/apos/components/AposTiptapLink.vue

```diff
@@ -9,7 +9,13 @@
       :icon="tool.icon ? tool.icon : false"
       :modifiers="['no-border', 'no-motion']"
     />
+    <!--
+      TODO: Confirm if the `editor` object passed into this `editor-menu-bubble`
+      should be the same one from the parent, or a fresh one. It does not seem
+      to use any of the editor configurations from the parent. - AB
+     -->
     <editor-menu-bubble
+      v-if="active"
```

Comment:
This being present before active immediately triggered an error in the console when a RTE was added to the page.